stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -2,39 +2,66 @@ import React, { useEffect, useState } from "react";
 import Components from "stockflux-components";
 import "./SecuritiesTable.css";
 import { Link } from "react-router-dom";
```

Comment:
Is this import used anywhere?

Reply:
No, was used in the table body. Has been removed now